Readme.md

```diff
@@ -104,6 +104,9 @@ When _elm-make_ encounters a compile error, we keep _elm-live_ running and give
 #### `--pushstate`
 Serve `index.html` on 404 errors. This lets us use client-side routing in Elm. For instance, we can have a URL like `http://localhost:8000/account` get handled by the Elm _navigation_ package instead of failing with a 404 error.
 
+#### `--before-build`
+Run a command before a build is run. This allows us to call external build tools that could generate Elm code, for example.
```

Comment:
I agree about mentioning elm-css here. I, too, think it's going to be the tool used most often with `--before-build`
I actually don't know any specific Elm code generators right now, which probably means I should be removing that from the documentation.
I mentioned it here because whenever e.g. quasiquoting comes up in Elm feature discussions, the counter point is made that one could generate Elm code with an external tool instead.
For example a templating language that compile to a view module etc.
Since this is not the case right now I'll remove that mention for now.

Also, if Elm tooling moves that direction in the future, elm-live would need to be able to watch additional files anyway, so it wouldn't work too well with the current implementation.